### ISSUE TITLE
Screwdriver spam is OP

### ISSUE THREAD

**Issue by freerealestate** [tgstation/-tg-station#10392]
Has been for years.

For those who don't know, aiming at the eyes with a screwdriver causes blindness, the victim to drop their weapon, and stuns, as well as the regular brute damage.

**Comment by PKPenguin321** [comment]
oh fuck not this again
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=3151
last time this 'issue' popped up on the github it got closed as a non issue and people said they wanted to keep this

**Comment by WJohn** [comment]
Definitely still deserves another design review. It's the only item in the game that stuns you due to "pain", specifically acts uniquely if aiming for the eyes, and causes blindness. None of the other items in the game can do this, outside of chems or flashing someone to oblivion.

**Comment by Marflowerose** [comment]
It also has the added huge benefit of not making any sound when used, at least when I last stabbed someone in the eyes, so you can be making ocular kebab right next door to someone without anyone being any wiser, except for the occasional fall sound.
Also, you can't stab someone in the eyes if they have any sort of glasses so only non-powergamey greyshirts get their eyes gauged out seeing as pretty much all actual jobs have some kind of glasses.

**Comment by freerealestate** [comment]
This post sums up the problems with it well, as well as providing a non-snowflakey solution:
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=3151#p76172

**Comment by SconesC** [comment]
No sound

No animation

All pain

**Comment by PKPenguin321** [comment]
those are bugs, the stun and blinding are balance
fix the bugs, leave the balance

**Comment by freerealestate** [comment]
It should really just act like a regular melee weapon, with the added bonus
of adding eye damage (though not as much as it currently does, because that
shit's a pain in the ass to fix) if you're aiming for the eyes. The stun is
complete bullshit and makes it far too easy to chainstun people on the
ground and crit them, with no counterplay available to them. It's the
choice of weapon of griefers everywhere.

**Comment by PKPenguin321** [comment]
if you have balance concerns post in the feedback forum, the issues section is supposed to be for bugs

**Comment by freerealestate** [comment]
The feedback forum that is probably barely looked at by most coders where
threads are mobbed with players shouting "git gud" to save their favourite
OP metastrats.

**Comment by freerealestate** [comment]
Balance is an important part of game design, so I'm not sure why it can't
be discussed here.

**Comment by Aranclanos** [comment]
Please use the feedback forums

**Comment by freerealestate** [comment]
I doubt you even read the feedback forums.

**Comment by freerealestate** [comment]
Much less make changes based on the suggestions there.

**Comment by Iamgoofball** [comment]
fuck off with that shit m8 you know shitting up the issue tracker with balance and feature request shit is dumb

**Comment by freerealestate** [comment]
It's the only effective way of making coders aware of problems with the
game, so I don't think it's dumb at all. I think it's integral to the
game's development.

**Comment by Iamgoofball** [comment]
issue tracker is for shit like 
"MAN THIS SHIT YOU ADDED ALLOWS ME TO KILL THE ENTIRE PLAYERBASE WITH 1 BUTTON CLICK I FEEL IT'S KINDA OP"
not
"I DIED PLS NERF"

Although, we really should look at eye code, it's horrible

**Comment by freerealestate** [comment]
But this is objectively OP and has been for years. It's been said better
than me the various reasons why the design is problematic and how exactly
it's unbalanced.

**Comment by Iamgoofball** [comment]
Then make a feedback thread

**Comment by freerealestate** [comment]
I would, but there's no reason not to be able to discuss it here.

**Comment by Iamgoofball** [comment]
Yes there is, the issue tracker is for bugs/extreme extreme problems.

**Comment by duncathan** [comment]
Issue: screwdrivers are snowflakey.

**Comment by SconesC** [comment]
@Aranclanos This is really if anything an issue report about the code being shit and there being 2 major bugs so

Reopen it maybe?

In case you can't be asked to read things before you close them (lmao) I'm referencing the fact that they make no attack animation and no sound unlike literally every other weapon in the game

**Comment by freerealestate** [comment]
I'll even change the issue report to reflect this, if it helps getting it
reopened it.

**Comment by PKPenguin321** [comment]
not a code problem = not our problem

to the forums with you, if you're scared of taking it there because "oh no the forums will disagree with me" then guess what, it's not gonna change anyways because the forums disagree with you

**Comment by freerealestate** [comment]
The problems it poses from a code perspective have already been explained
to you.

**Comment by PKPenguin321** [comment]
then make an issue regarding the code aspects (it not having animation, hitsound)

**Comment by freerealestate** [comment]
I already did. In future, make sure the ground you're standing on is firm before you start acting condescending.

**Comment by Iamgoofball** [comment]
That's acceptable now.

**Comment by freerealestate** [comment]
So is this going to be reopened now, or is everyone too afraid of Aranclanos turning on them?

**Comment by Iamgoofball** [comment]
can you rephrase that so that you're not being a bigger dick then I am

**Comment by freerealestate** [comment]
Even if I did, you'd still only be getting silver in the dick olympics.
